undertow/src/main/java/org/wildfly/extension/undertow/HttpsListenerResourceDefinition.java

```diff
@@ -60,14 +60,14 @@
             .setCapabilityReference(SSL_CONTEXT_CAPABILITY, LISTENER_CAPABILITY_NAME, true)
             .setFlags(AttributeAccess.Flag.RESTART_ALL_SERVICES)
             .setValidator(new StringLengthValidator(1))
-            .setAccessConstraints(SensitiveTargetAccessConstraintDefinition.SECURITY_REALM_REF)
```

Comment:
+1 We have a separate RFE to address that one, wanted to keep it separate so we can discuss it as a single unit.  

Will follow up with that one once the management PR is in place as at that point the majority of references will be completed.